### PR TITLE
docs: add warning about dynamic import of css

### PR DESCRIPTION
**Pull request by InerkyJad**
The warning is regarding the use of dynamic imports inside of the script tag, it's related to #21803

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by MuhammadM1998** [comment]
Hello,
Maybe you can add an example of a static import ( `import "./style.css`) too as the current line maybe confusing.

Is this seems better?
```
Make sure to use a **synchronous** import `import "./style.css` instead of a **dynamic** import `import("~/assets/style.css")` for server-side compatibility.
```

**Review by florian-lefebvre — CHANGES_REQUESTED** [review]
I agree with @MuhammadM1998, adding an example of synchronous import is less confusing. Could you add it?

**Comment by florian-lefebvre** [comment]
@Hebilicious can you review?

**Comment by danielroe** [comment]
Maybe add this as a comment to the existing example of synchronous import above?

**Review by danielroe — APPROVED** [review]
Thank you - perfect!

File: docs/1.getting-started/4.styling.md [hunk 1]
```diff
@@ -18,7 +18,11 @@ You can use a javascript import, or a css [`@import` statement](https://develope
 
 ```vue [pages/index.vue]
 <script>
+// Use a static import for server-side compatibility
 import '~/assets/css/first.css'
+
+// Caution: Dynamic imports are not server-side compatible.
```
Comment (florian-lefebvre) [inline docs/1.getting-started/4.styling.md:24]:
```suggestion
// Caution: Dynamic imports are not server-side compatible
```
To keep it consistent with the previous comment

**Review by florian-lefebvre — CHANGES_REQUESTED** [review]

**Pull request merged**